res/css/structures/_RoomStatusBar.scss

```
@@ -128,23 +128,20 @@ limitations under the License.
                 mask-repeat: no-repeat;
                 mask-position: center;
                 mask-size: contain;
+                width: 18px;
+                height: 18px;
+                top: calc(50% - 9px); // text sizes are dynamic
```

I assume that `9px` is a reference to `18px / 2`
To avoid a mismatch between that value and the `height` you can

```suggestion
                top: 50%; // text sizes are dynamic
                transform: translateY(-50%);
```

Which will always be based off the height of the element and not the parent height